Gems/Atom/Tools/AtomToolsFramework/Code/Source/Viewport/RenderViewportWidget.cpp

```diff
@@ -146,11 +146,16 @@ namespace AtomToolsFramework
         if (auto existingScene = scene->FindSubsystem<AZ::RPI::ScenePtr>())
         {
             m_viewportContext->SetRenderScene(*existingScene);
-            if (auto auxGeomFP = existingScene->get()->GetFeatureProcessor<AZ::RPI::AuxGeomFeatureProcessorInterface>())
+
+            // return if only a render pipeline for the viewport exist. Otherwise we need to continue to ensure render pipeline installed for the scene.
```

Comment:
The language on this could be cleaned up a little, maybe:
```suggestion
            // If we have a render pipeline, use it and ensure an AuxGeom feature processor is installed.
            // Otherwise, fall through and ensure a render pipeline is installed for this scene.
```